Vagrantfile

```diff
@@ -5,7 +5,7 @@ Vagrant.configure('2') do |c|
   server_post_up = <<-EOS.gsub(/^[[:blank:]]{4}/, '')
     Now run the following commands to get going:
 
-        simp config -a /vagrant/answers.yaml
+        simp config -a /vagrant/simp_config_answers.yaml
```

Comment:
Alright but this provided file will be different than what a user might get when they look in their generated simp_conf.yaml.